README.md

```diff
@@ -1,4 +1,114 @@
-News and ideas
---------------
+## Good practices
 
-Repository of 3mdeb blog posts.
+### Grammarly 
+
+Grammarly is a great, free tool for all bloggers and anyone who needs to write documentation in English.
+It will let you know if you skipped a coma or made a typo, as well, as it will check advanced grammar mistakes, too. Bear in mind, that the free version has its limits, so you need to keep an eye on it at all times and still, you are the one who distinguishes when to use a/an or the, as it only suggests changes.
+
+Two versions of Grammarly are available: a plugin for Chrome/Chromium or online application. You need to create an account (it's for free) to be able to use Grammarly.
+
+Visit the website [Grammarly](https://app.grammarly.com/) and create an account.
+
+>It is a MUST-HAVE application for anyone who writes posts or documentation, so feel obliged to use it.
+
+---
+
+### Images optimization
+
+When you write an article and want to add some photos, you need to remember about optimizing your photos. If you don't do that, your readers will need to download heave images in order to read your articles. Also, that impacts our score on PageSpeed Insights. Besides, without losing any quality, you can reduce (for example) a file size from 2MB to 300KB, which is an enormous difference.
+
+If you want a photo to be optimized, you can simply send it to me (Michał Ścibisz), or do it by yourself, if only you have access to Wordpress.
```

Comment:
@mscibisz avoid using the name it may be outdated soon, but this name will stay here. Please use term 3mdeb Wordpress Administrator so people can ask on the internal Slack channel.